### ISSUE TITLE
Design suggestions for /numbers

### ISSUE THREAD

**Issue by jf1** [mysociety/yournextrepresentative#885]
Currently an election is displayed as:
![image](https://cloud.githubusercontent.com/assets/4711363/14938885/07ee7ab8-0f2b-11e6-93a3-cde33cec2ee8.png)
with several clicks required to see more numbers.

This could be displayed as:
**Statistics for the [Adur local election](https://candidates.democracyclub.org.uk/election/local.adur.2016-05-05/constituencies)**
65 candidates representing 7 [parties](https://candidates.democracyclub.org.uk/numbers/election/local.adur.2016-05-05/parties) are standing in 14 [elections](https://candidates.democracyclub.org.uk/numbers/election/local.adur.2016-05-05/posts). All candidate lists are locked,

I used 'elections' rather than 'posts' as one of these elections has two vacancies, but am not precious about the final wording - the main desire/suggestion is to see all of the available numbers and reduce the space required for each election. Hopefully there's already code to insert singular candidate or post text when there's only one of them, and obviously something like "1 candidate list is unlocked" etc if it's > 0.

On the [election's parties page](https://candidates.democracyclub.org.uk/numbers/election/local.adur.2016-05-05/parties) I don't see any benefit in listing parties that aren't fielding any candidates:
![image](https://cloud.githubusercontent.com/assets/4711363/14939021/0e259106-0f2f-11e6-9eae-3327a75eafa7.png)

And [each party's page for an election](https://candidates.democracyclub.org.uk/election/local.adur.2016-05-05/party/party:52/conservative-and-unionist-party) would read better if the Electoral Commission text was displayed below the candidate. There are also a couple of anomalies with the list icons:
![image](https://cloud.githubusercontent.com/assets/4711363/14938981/b0349e80-0f2d-11e6-9cea-ae8ac0d934a0.png)

**Comment by wfdd** [comment]
I'm not sure if converting the election info into prose would be an improvement; I'd rather keep it as a list 'cause it's easier to scan that way, but maybe space it horizontally?

**Comment by wfdd** [comment]
I'm not sure if converting the election info into prose would be an improvement; I'd rather keep it as a list 'cause it's easier to scan that way, but maybe space it horizontally?

In the last picture, candidates are grouped by post.  The hollow circles indicate a sublist.  There's two candidates for Widewater, which is why there's no solid circle on the left of Pigott.  There's no hollow circle in Marine 'cause there's no candidates standing there, and hence no sublist.

But these lists don't (visually) translate very well to party lists either; I think it'd be better if we shed 'is standing in _x_' and made each post a subheading or the outer bullet point.

**Comment by tfgg** [comment]
Yeah, I'm not a fan of the current lists, but I'm not sure prose is that great either. Wouldn't it be best as a table?

**Comment by jf1** [comment]
No need for it to be prose, the suggestion is primarily to display all those numbers, ideally on one line.

A table could work well but I doubt (all of) each election name could easily fit on the same line, especially for long ones eg "Cambridgeshire by-election: St. Neots Eaton Scon & Eynesbury ward"

Thanks for explaining the hollow and solid circles - I didn't realise they were done deliberately and had those meanings - and am not convinced they add anything! (I did look at others, but in the same election so they all had no solid circle on their last line).

FWIW I'm fine with the 'is standing in x' text and don't see any benefit in changing that - I certainly wouldn't want the list to become twice as long unless it was displaying more information.

**Comment by wfdd** [comment]
> FWIW I'm fine with the 'is standing in x' text and don't see any benefit in changing that - I certainly wouldn't want the list to become twice as long unless it was displaying more information.

It's overwhelming with party lists and large districts, like [here](http://neavouli.eu/election/vouleftikes-ekloges-2016/party/diko/dimokratiko-komma).  There's gotta be some way to accommodate both use cases.  I don't think there's enough contrast between posts there either; the solid circles are easily missed.

**Comment by jf1** [comment]
I agree busy pages like that aren't ideal, but don't think the 'is standing in _x_' text causes the problem.

[This](https://candidates.democracyclub.org.uk/election/sp.r.2016-05-05/party/party:102/scottish-national-party-snp) is similar, I'd suggest: "CandidateA, CandidateB and CandidateC are standing in AreaD" etc.

An alternative would be a bold line with eg: "**Standing in AreaD are:**" then CandidateA, CandidateB and CandidateC etc on separate lines underneath - but this display wouldn't be apt for most of the current UK elections so I think there'd need to be a per-election setting for those where >5 candidates from one party are expected in each post.

ps @mhl just fyi, the candidates aren't sorted alphabetically in [this](https://candidates.democracyclub.org.uk/election/sp.r.2016-05-05/party/party:102/scottish-national-party-snp).

**Comment by wfdd** [comment]
This is what I've done for the Cypriot election:

![screencapture-neavouli-eu-election-vouleftikes-ekloges-2016-party-simaia-koinoniko-kinima-simaia-1462099572855](https://cloud.githubusercontent.com/assets/8489551/14941431/5c2a041c-0fa3-11e6-8a68-9fb6d8d890b5.png)

Because of the columns, this is probably gonna take up about equal space as the current layout in FPTP elections on the desktop.  It takes up significantly less space with party lists.

**Comment by jf1** [comment]
That looks like an improvement @wfdd although I'd sort by the party list positions and put those numbers on the left of the candidate names.

This layout might also work well for elections with multiple vacancies but no party lists, such as next year's UK County Council elections.

Merely as I can't recall seeing it elsewhere on the site, is the use of two columns fine?
